plugin.video.vtm.go/resources/lib/vtmgo/vtmgoepg.py

```diff
@@ -78,6 +82,8 @@ class VtmGoEpg:
     """ VTM GO EPG API """
     EPG_URL = 'https://vtm.be/tv-gids/api/v2/broadcasts/{date}'
 
+    EPG_NO_BROADCAST = 'Geen uitzending'
```

Comment:
Needs to be localised

Reply:
No, this is the exact string we get from the EPG api to indicate that there is no data. This isn't shown in the UI.